### PR TITLE
Fixed design of apply for master button on dashboard

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
Fixes https://github.com/mitodl/micromasters/issues/373

#### What's this PR do?
- Fixed design of apply for master button of the program in dashboard.

#### Where should the reviewer start?
- CourseList.js

#### How should this be manually tested?
- Open dashboard and see at the end of a program.

#### Screenshots (if appropriate)
@pdpinch @aliceriot @noisecapella 
screen shot 2016-05-25 at 6 50 13 pm

### CODE REVIEW

File: static/js/components/CourseList.js [hunk 1]
```diff
@@ -112,8 +112,10 @@ class CourseList extends React.Component {
           <div className="apply-for-ms">
             <br/>
             <p>You need to pass {totalCourses - coursesPassed} more courses
-              before you can apply for <strong>{program.title}</strong> Master's Degree.</p>
-            <Button bsStyle="primary" disabled>Apply for <strong>{program.title}</strong> Master Degree</Button>
+              before you can apply for <strong>{program.title}</strong> Master’s Degree.</p>
```
Comment (pdpinch) [inline static/js/components/CourseList.js]: Copy here needs an article. 

```
before you can apply for the <strong>{program.title}</strong> Master’s Degree.</p>
``

File: static/scss/dashboard.scss [hunk 2]
```diff
@@ -287,3 +287,15 @@ c.validation-wrapper {
 .header-privacy-tab {
   color: #A2AAB3;
 }
+
+button.btn-apply-for-ms {
+  font-family: "Open Sans";
+  background: #B2C9D2;
```
Comment (pdpinch) [inline static/scss/dashboard.scss]: This color is only for the disabled state. I know we don't show the button enabled yet, but doesn't it make sense to put this in the disabled style?

**Comment by Ferdi** [comment]
I'm afraid that wont pass accessibility requirements - low contrast.   Can you please look at Material Design guidelines / examples ?

**Comment by pdpinch** [comment]
Good point Ferdi. I wasn't watching that. 

However, as far as I can tell, the mdl disabled button is low contrast too. See https://www.google.com/design/spec/components/buttons.html#buttons-raised-buttons

This led me to this [stackoverflow question](http://ux.stackexchange.com/questions/80881/do-disabled-buttons-still-need-to-be-contrast-compliant-for-accessibility), which points to the [relevant w3c standard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html):

> Text or images of text that are part of an inactive user interface component... have no contrast requirement.

Should we go ahead and use the mdl-recommended grey button with grey text? Or leave it as is?

**Comment by amir-qayyum-khan** [comment]
@pdpinch we can also look into material-ui buttons.
please see this as well.
http://www.material-ui.com/#/components/raised-button

**Comment by pdpinch** [comment]
The material-ui treatment for disabled buttons is the same as the Google material design guidelines. Go with whichever is easier for you to implement. 

I'll check with our accessibility director, but we should proceed.

**Comment by amir-qayyum-khan** [comment]
should i use same color i.e grayish text that is in the material-ui example?

**Comment by pdpinch** [comment]
Yes. You can use the same gray colors from the material-ui example for now.

**Comment by amir-qayyum-khan** [comment]
@pdpinch made the change and button looks like this.

screen shot 2016-05-28 at 12 30 54 am

File: static/js/components/CourseList.js [hunk 3]
```diff
@@ -88,6 +89,12 @@ class CourseList extends React.Component {
         </div>;
       });
 
+      let applyForMSBtnLabel = `Apply for the ${program.title} Master’s Degree`;
+      const btnStyle = {
+        'text-transform': 'none',
+        'padding': '25px 16px'
+      };
```
Comment (pdpinch) [inline static/js/components/CourseList.js]: Why is this necessary?

Comment (amir-qayyum-khan) [inline static/js/components/CourseList.js]: if i not use text-transform the all text will be capitalize.

**Comment by amir-qayyum-khan** [comment]
@gsidebo @pdpinch please review

**Comment by pdpinch** [comment]
👍
